src/report/utility-reports/hello-world.scm

```diff
@@ -354,9 +354,9 @@ or extending existing reports.")))
 new, totally cool report, consult the mailing list %s.")
           (gnc:html-markup-anchor 
            "mailto:gnucash-devel@gnucash.org"
-           (gnc:html-markup-tt "gnucash-devel@gnucash.org")))
-         (_ "For details on subscribing to that list, see &lt;http://www.gnucash.org/&gt;.")
-         (_ "You can learn more about writing scheme at &lt;http://www.scheme.com/tspl2d/&gt;."))
+           (gnc:html-markup-tt "gnucash-devel@gnucash.org"))
+          (_ "For details on subscribing to that list, see &lt;http://www.gnucash.org/&gt;. ")
+          (_ "You can learn more about writing scheme at &lt;http://www.scheme.com/tspl2d/&gt;.")))
```

Comment:
This time I have applied your patch and ran some tests.

The change above appears to do different things for me than for you. Before your patch, I have this:

> For help on writing reports, or to contribute your brand new, totally cool report, consult the mailing list gnucash-devel@gnucash.org .For details on subscribing to that list, see <http://www.gnucash.org/>.You can learn more about writing scheme at <http://www.scheme.com/tspl2d/>.

After your patch I get this:

> For help on writing reports, or to contribute your brand new, totally cool report, consult the mailing list gnucash-devel@gnucash.org .

The rest of the text is gone completely. Can you check this ?